src/services/DateService.ts

```diff
@@ -300,8 +299,11 @@ export class DateService {
       // }
     }
 
-    if (!stop && !shouldStopOnLargest2(largestTwoCounter) && resolution <= TimespanResolution.seconds) {
-
+    if (!stop && !shouldStopOnLargest2(largestTwoCounter) && ((seconds ||
```

Comment:
@hiaux0  Can you please review this?  I thought that code was already working.

Reply:
@dkent600 
As discussed, separate ticket 
https://github.com/PrimeDAO/prime-deals-dapp/issues/469

Reply:
Can we revert this now that the bug is fixed elsewhere?